db/migrate/20240111200053_create_system_uptimes.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSystemUptimes < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
I did not test this yet, can you check if this migration is able to be rolled back by default? PubCloud needs to be able to rollback in case of regressions.

Reply:
vagrant@rmtleap155:~/Code/rmt> rake db:rollback RAILS_ENV=test
== 20240111200053 CreateSystemUptimes: reverting ==============================
[strong_migrations] DANGER: Lock timeout is longer than 0 seconds: 86400
-- remove_foreign_key(:system_uptimes, :systems, {:column=>:system_id, :validate=>false})
   -> 0.0058s
-- remove_index(:system_uptimes, [:system_id, :online_at_day, :online_at_hours], {:unique=>true, :name=>"id_online_day_hours"})
   -> 0.0078s
-- drop_table(:system_uptimes, {:id=>false})
   -> 0.0027s
== 20240111200053 CreateSystemUptimes: reverted (0.0251s) =====================